core-services/sshd-bastion/telco/deployment.yaml

```diff
@@ -77,4 +83,4 @@ items:
             secretName: sshd-host-keys
         - name: config
           configMap:
-            name: sshd-config
\ No newline at end of file
+            name: sshd-config
```

Comment:
no need for that \n

Reply:
vim is set to do that for me to follow POSIX standards